azure-pipelines-with-e2etests-teamsV1.yml

```diff
@@ -0,0 +1,80 @@
+# This pipeline checks-out current client sdk repo and stable app hosting sdk repo (as specified in AppHostingSdkGitRef).
```

Comment:
Is there a reason we'd want to create a new yml file for this process? I'm currently working on combining all of our yaml files and it seems a lot of the content here is duplicated from our other actions. (It would be a lot more efficient not running them twice if we don't need to.)

Reply:
This pipeline is for running e2e tests running against TeamsJs V1. The reason we want it to run on every PR is to make sure changes made to hub SDK don’t break the backwards compatibility.

Reply:
We can still run it on every PR without creating another file. We could maybe add steps to an existing file like azure-pipelines-with-e2etests.yml?

Reply:
You are right, thanks. To avoid messing current pipeline, I will delete the file for now and make the change in azure-pipelines-with-e2etests.yml later or maybe in other PR.

Reply:
It looks like just this task needs to be included to run the test, maybe you can just put it in the file now? (You're free to do it whenever you see fit tho ofc)

` - bash: 'node tools/cli/runAppsWithE2ETests.js --envType=teamsV1'
    displayName: 'Run E2E integration tests'
    condition: succeeded()
    workingDirectory: '$(AppHostingSdkProjectDirectory)'`

Reply:
This is a tradeoff. On one hand, if we create a new pipeline, it'll force us to clone, build and start the app before running the tests. If we were to add it to the existing pipeline we could reuse the fact that the app is already running. On the other hand, with a separate pipeline all this would happen in parallel to the existing pipeline. If we add it to the existing one, it'll be sequential.

Reply:
If the reason to have them in a single pipeline is to avoid duplicating all the steps, we could accomplish it via stage templates: https://docs.microsoft.com/en-us/azure/devops/pipelines/yaml-schema?view=azure-devops&tabs=example%2Cparameter-schema#stage-templates

Reply:
Actually, looks like the build command is different between these (`build-test-app` vs `build-test-app-V1`, so yeah, I think we should have them as separate pipelines, and maybe consider using stage templates.

Reply:
Thanks for catching that I missed the build command for the test app here being different though ofc!

Reply:
As discussed offline I'll be going ahead with this file and this staging work can be tabled to another PR